src/compiler/diagnosticMessages.json

```diff
@@ -1859,10 +1859,22 @@
         "category": "Error",
         "code": 2553
     },
-    "Expected {0} type arguments, but got {1}.": {
+    "Expected at least {0} arguments, but got {1}.": {
         "category": "Error",
         "code": 2554
     },
+    "Expected {0} arguments, but got a minimum of {1}.": {
+        "category": "Error",
+        "code": 2555
+    },
+    "Expected at least {0} arguments, but got a minimum of {1}.": {
+        "category": "Error",
+        "code": 2556
+    },
+    "Expected {0} type arguments, but got {1}.": {

```

Comment:
In some cases, type arguments also need "at least" some number. Is that something that would be done in a different PR?